### DISCUSSION TITLE
Modulated signal, "shadow trace"

### DISCUSSION THREAD

**Discussion by pietss** [egzumer/uv-k5-firmware-custom#305] [General]
In the meantime I add a third observation.
In spectrum mode, calibrated squelch, a modulated signal arrives. At the end of the transmission it is necessary to raise the squelch level because a "shadow trace" remains (visible with the bar that has risen compared to the adjacent frequencies), so if they subsequently arrive other low intensity modulated signals would not be detected by having to raise the squelch bar.
I apologize for my artisanal explanations....

**Comment by ErikS-web** [comment]
Perhabs you can take some pictures and insert them here in the comment-field
